types/setting.go

```diff
@@ -1172,6 +1185,8 @@ func ValidateSetting(name, value string) (err error) {
 		if value < 0 {
 			return fmt.Errorf("value %v should be positive", value)
 		}
+	case SettingNameStorageReservedPercentage:
```

Comment:
Error out if the value is negative.

Reply:
`fallthrough` to `SettingNameStorageMinimalAvailablePercentage`
It will return error if value is negative, does this what you point out?

Reply:
Ah, yeah. Thank you.